src/parser/parser_flow.ml

```diff
@@ -150,6 +153,7 @@ module rec Parse : PARSER = struct
     | _ when Peek.is_class env -> class_declaration env decorators
     | T_INTERFACE -> interface env
     | T_DECLARE -> declare env
+    | T_IMPORT when Peek.token ~i:1 env = T_LPAREN -> expression env
```

Comment:
I think it should be handled in `statement`, it's probably just enough to remove `T_IMPORT` there